apps/storefront/src/utils/basicConfig.ts

```diff
@@ -8,7 +8,7 @@ export const {
 const { VITE_IS_LOCAL_ENVIRONMENT } = import.meta.env;
 
 const generateBcStorefrontAPIBaseUrl = () => {
-  if (VITE_IS_LOCAL_ENVIRONMENT === 'TRUE') return '/bigcommerce';
+  if (VITE_IS_LOCAL_ENVIRONMENT === 'TRUE') return window.origin;
```

Comment:
maybe this is not needed anymore.

Reply:
I followed your suggestion, you're right, it not needed